BuildingBlocks/src/BuildingBlocks.Infrastructure/Persistence/BlobStorage/GoogleCloudStorage/GoogleCloudStorageServiceCollectionExtensions.cs

```diff
@@ -47,6 +48,9 @@ public static void AddGoogleCloudStorage(this IServiceCollection services, Googl
 
             return new GoogleCloudStorage(storageClient, logger);
         });
+
+        services.AddHealthChecks()
+            .Add(new HealthCheckRegistration("Google Cloud", sp => new GoogleCloudStorageHealthCheck(sp.GetRequiredService<StorageClient>(), options), HealthStatus.Unhealthy, null));
```

Comment:
The health check shouldn't be called "Google Cloud", because
1. It doesn't check the health of the whole Google Cloud
2. It should be transparent to the caller whether we use Google Cloud Storage, Azure Storage Account, or whatever.

So just call it "blob_storage".